voice/voice-api/transfer-call-inline-ncco.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+source "../../config.sh"
+source "../../jwt.sh"
+
+curl -X POST https://api.nexmo.com/v1/calls/$UUID \
+  -H "Authorization: Bearer "$JWT \
+  -H "Content-Type: application/json"\
+  -d '{"action": "transfer",
+      "destination": {"type": "ncco", "ncco": ["action":"talk", "text":"This is a transfer action using an inline NCCO"]}}'
```

Comment:
```suggestion
      "destination": {"type": "ncco", "ncco": [{"action":"talk", "text":"This is a transfer action using an inline NCCO"}]}}'
```
The NCCO is an array of objects.

Reply:
Also fixed. My bad.